### PR TITLE
Add halloween theme

### PR DESCRIPTION
**Pull request by glacambre**
I wrote a very spooky theme for halloween but I'm currently encountering two bugs and I can't seem to fix them, I was hoping you'd have ideas.
The first bug is that the commandline is resized to 1px when there are no completions available.
The second bug is https://github.com/tridactyl/tridactyl/issues/581 .

If we can't find ways to fix these bugs I think it'd be useful to keep them in mind when the UI rewrite happens.

### CODE REVIEW

**Comment by bovine3dom** [comment]
Updated priority due to time-sensitivity.

**Comment by bovine3dom** [comment]
The cause must be that `cmdline_iframe.contentWindow.document.body.offsetHeight` is 1px high when there are no completions in your theme. It needs to be the same height as the input box.

The line in question: https://github.com/tridactyl/tridactyl/blob/master/src/content/commandline_content.ts#L61

I do not know _why_ the offsetHeight is 1px in your theme but no others. If we can figure that out we'll be most of the way there to fixing it.

File: src/static/themes/halloween/halloween.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
+
+.TridactylThemeHalloween #cmdline_iframe {
+    position: fixed !important;
```
Comment (bovine3dom) [inline src/static/themes/halloween/halloween.css]:
```suggestion
     order: 1;
```

File: src/static/themes/halloween/halloween.css [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  background: white !important;
+  margin: 4px;
+  border: solid black 1px;
+}
+
+html.TridactylOwnNamespace.TridactylThemeHalloween, .TridactylThemeHalloween #completions {
+  background: #bfb8bf;
+}
+
+.TridactylThemeHalloween #completions .sectionHeader {
```
Comment (bovine3dom) [inline src/static/themes/halloween/halloween.css]:
```suggestion
.TridactylThemeHalloween #completions .sectionHeader {
    order: 2;
```

**Comment by bovine3dom** [comment]
The suggestions I've made haven't fixed it entirely but hopefully they'll send you in the right direction.

**Comment by glacambre** [comment]
Done, thanks for your help, `order: 1` and `order: 2` indeed helped fixing everything.

I'm note sure what looks the best here: https://github.com/tridactyl/tridactyl/pull/1100/files#diff-58ce7236039782718318a60603f547e4R90 , feel free to make a choice and edit things before merging this PR.

**Comment by cmcaine** [comment]
🎃

**Comment by bovine3dom** [comment]
> cmcaine merged commit

Scariest thing I've seen all day

**Pull request merged**